### PR TITLE
feat(sql): added `json-sql` hub publish metadata

### PR DESCRIPTION
**Pull request by galibey**

### CODE REVIEW

File: smartmodules/json-sql/SmartModule.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+group = "infinyon"
+version = "0.1.0"
+apiVersion = "0.1.0"
+description = "JSON to SQL transformation Smartmodule"
+license = "Apache-2.0"
+repository = "https://github.com/infinyon/fluvio"
+
+
+[params]
+with = { description = "Transformation mapping" }
```
Comment (sehz) [inline smartmodules/json-sql/SmartModule.toml:12]: shouldn't this be something like `mapping`?

**Review by sehz — CHANGES_REQUESTED** [review]

File: smartmodules/json-sql/SmartModule.toml (hunk 1 above)
Comment (galibey) [reply smartmodules/json-sql/SmartModule.toml:12]: I agree that `mapping` now feels more descriptive, but the change will also affect the SQL connector. Should we do it here?

Comment (sehz) [reply smartmodules/json-sql/SmartModule.toml:12]: Probably both

Comment (galibey) [reply smartmodules/json-sql/SmartModule.toml]: Done

**Review by sehz — APPROVED** [review]
LGTM

**Comment by superstator** [comment]
LGTM

**Pull request merged**